manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -155,6 +157,9 @@ spec:
         hostPath:
           path: /var/run/ztunnel
           type: DirectoryOrCreate # ideally this would be a socket, but ztunnel may not have started yet.
+      # pprof needs a writable /tmp, and we don't have that thanks to `readOnlyRootFilesystem: true`, so mount one
```

Comment:
On envoy we are dumping to `/var/lib/istio/data`. That seems to make sense for consitency vs `/tmp`? And tmp is more likely to be accidentally used by something else... although nothing should be writing to the filesystem, I suppose

Reply:
This would require a change in ztunnel

Reply:
> On envoy we are dumping to /var/lib/istio/data. That seems to make sense for consitency vs /tmp? And tmp is more likely to be accidentally used by something else... although nothing should be writing to the filesystem, I suppose

Right - I am inclined to leave it as just `/tmp` - nothing else is running here, the rest of the FS is R/O, and the odds that something else writes on top of a ztunnel pprof file is nil.


We could change ztunnel to make it configurable but in this case it's really not a big deal, IMO.